docs/vmtype.md

```diff
@@ -0,0 +1,35 @@
+# vmType
+
+Lima supports two ways of running guest machines:
+- [qemu](#qemu)
+- [vz](#vz)
+
+## QEMU
+"qemu" option makes use of QEMU to run guest operating system. 
+This option is used by default if "vmType" is not set.
+
+## VZ
+"vz" option makes use of native virtualization support provided by macOS Virtualization.Framework.
+
+An example configuration:
+```yaml
+# Example to run ubuntu using vmType: vz instead of qemu (Default)
+vmType: "vz"
+images:
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+mounts:
+  - location: "~"
+mountType: "virtiofs"
+```
+
+### Caveats
+- "vz" option is only supported on macOS 13 or above
+- Virtualization.framework doesn't support running "intel guest on arm" and vice versa
+
+### Known Issues
+- "vz" doesn't support `legacyBoot: true` option, so guest machine like centos-stream, archlinux, oraclelinux will not work
+- Host to guest networking (`networks` section in lima yaml) is not supported
```

Comment:
Is this outrightly not supported by the driver, and with no workarounds?

Reply:
Expected to be added as "vznat" network in a separate PR
https://github.com/lima-vm/lima/pull/1147#discussion_r1016143136